packages/cli-ui/src/components/ProjectManager/ProjectList.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Project } from '../../pages/Projects'
+import ProjectListItem from './ProjectListItem'
+import { useTranslation } from 'react-i18next'
+import css from './style.module.scss'
+
+interface ProjectList {
+      projects: Project[];
+      favorits: Project[];
+      onDelete(id: number, favorite: boolean): void;
+      onFavorite(id: number, favorite: boolean): void;
+}
+
+export default function ProjectList ({ favorits, projects, onDelete, onFavorite }: ProjectList) {
```

Comment:
Correctly to use favorites
```suggestion
export default function ProjectList ({ favorites, projects, onDelete, onFavorite }: ProjectList) {
```